inference-engine/src/transformations/include/transformations/op_conversions/simplify_ctc_greedy_decoder_seq_len.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API SimplifyCTCGreedyDecoderSeqLen;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief SimplifyCTCGreedyDecoder converts v6:CTCGreedyDecoderSeqLen into v0::CTCGreedyDecoder.
+ *
+ *            data[N, T, C]    seq_len[N]
+ *                   \          /
+ *            CTCGreedyDecoderSeqLen
+ *
+ * will be converted to
+ *
+ *           data[T, N, C]   seq_mask[T, N]
+ *                    \         /
+ *                  CTCGreedyDecoder
+ *                    /         \
+ *       class_index[N, T]    seq_len[N]
+ *
+ * The transformation works only for case when the blank_index input is not provided as input to
```

Comment:
The statement " The transformation works only for case when the blank_index input is not provided as input" is not `True`. The transformation does't work if `blank_index != C-1`

Reply:
We cannot check blank_index != C-1 in transformation. 
We can just check have we blank_index input or not.